tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/plugin/StompPluginTest.java

```diff
@@ -127,10 +131,33 @@ public void testSendAndReceive() throws Exception {
                                                AFTER_CREATE_SESSION, BEFORE_CLOSE_SESSION, AFTER_CLOSE_SESSION, BEFORE_CREATE_CONSUMER,
                                                AFTER_CREATE_CONSUMER, BEFORE_CLOSE_CONSUMER, AFTER_CLOSE_CONSUMER, BEFORE_CREATE_QUEUE, AFTER_CREATE_QUEUE,
                                                MESSAGE_ACKED, BEFORE_SEND, AFTER_SEND, BEFORE_MESSAGE_ROUTE, AFTER_MESSAGE_ROUTE, BEFORE_DELIVER,
-                                               AFTER_DELIVER);
+                                               AFTER_DELIVER, BEFORE_ADD_ADDRESS, AFTER_ADD_ADDRESS);
+
+      } catch (Throwable e) {
+         fail(e.getMessage());
+      }
+
+   }
+
+   @Test
+   public void testStompAutoCreateAddress() throws Exception {
+
+      try {
+         URI uri = new URI("ws+v12.stomp://localhost:61613");
+         StompClientConnection newConn = StompClientConnectionFactory.createClientConnection(uri);
+         newConn.connect(defUser, defPass);
+
+         subscribeQueue(newConn, "a-sub", "autoCreated");
+
+         // unsub
+         unsubscribe(newConn, "a-sub");
+         newConn.disconnect();
+
+         verifier.validatePluginMethodsAtLeast(1, BEFORE_ADD_ADDRESS, AFTER_ADD_ADDRESS,
+               BEFORE_REMOVE_ADDRESS, AFTER_REMOVE_ADDRESS);
 
       } catch (Throwable e) {
-         e.printStackTrace();
+         fail(e.getMessage());
```

Comment:
the printStackTrace is useful when it fails :)
I have been there.

Reply:
this is actually the only request I'm going to make.. almost merged this.

Reply:
hmmmm.. but it was being ignored before?

maybe just remove the catch and let it fail? remove the throws Exception on the test?

Reply:
yeah i can remove it, i'm not sure why it's there at all

Reply:
ok i updated the PR